README.md

```diff
@@ -1,3 +1,7 @@
+# This is a fork!
```

Comment:
For the purpose of keeping a minimal diff from the source, you could probably delete this, especially since the Github ui shows that it's forked.